docs/batch-changes/configuring-credentials.mdx

```diff
@@ -2,14 +2,16 @@
 
 <p className="subtitle">Learn how to configure access tokens for code hosts when creating changesets.</p>
 
-Interacting with a code host (such as creating, updating, or syncing changesets) is made possible by configuring an access token for that code host. Sourcegraph uses these tokens to manage changesets on your behalf and with your specific permissions.
+Interacting with a code host (such as creating, updating, or syncing changesets) is made possible by configuring credentials for that code host. Sourcegraph uses these credentials to manage changesets on your behalf and with your specific permissions.
 
 ## Requirements
 
-- Sourcegraph instance with repositories in it. Read more in the [Quickstart](/batch-changes/quickstart) guide on how to set up a Sourcegraph instance
-- Account on the code host with access to the repositories you wish to target with your batch changes
+- Sourcegraph instance with repositories in it. Read more in the [Quickstart](/batch-changes/quickstart) guide on how to set up a Sourcegraph instance.
+- Account on the code host with access to the repositories you wish to target with your batch changes.
 
-## Types of access tokens used by Batch Changes
+## Types of credentials used by Batch Changes
+
+Batch Changes can use access tokens for all code hosts, and [GitHub apps](#github-apps) for GitHub code hosts.
 
 Two types of access tokens can be configured to use with Batch Changes:
```

Comment:
These should be updated to reflect the switch from `access tokens` to credentials.

The concept is the same, we're just referring to it differently.

- Personal Credential
- Global service credential


cc @bahrmichael

Reply:
Should we maybe rename `Personal credential` to `User Credential`?

Reply:
Can do that, but then we have user in some places and personal in other places. Just something to keep in mind, but it's probably good to start the transition now.

Reply:
The other places we have `User credential` is more in the codebase right?
Not in the docs.

Reply:
I think so.

Reply:
Then that's fine. This doc is customer facing.

Reply:
I pushed an update. Let me know what you think.